dts/bindings/clock/fixed-clock.yaml

```diff
@@ -0,0 +1,41 @@
+#
+# Copyright (c) 2019 Derek Hageman <hageman@inthat.cloud>
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+---
+title: Generic fixed rate clock provider
+version: 0.1
+
+description: >
+    This is a representation of a generic fixed rate clock provider.
+
+inherits:
+    !include base.yaml
```

Comment:
I don't think we should have the inherits from base.yaml here.

Reply:
Ok, I've removed the inherit.